tests/dragonfly/replication_test.py

```diff
@@ -2407,9 +2407,17 @@ async def test_replicate_old_master(
 
     dfly_version = "v1.19.2"
     released_dfly_path = download_dragonfly_release(dfly_version)
-    master = df_factory.create(version=1.19, path=released_dfly_path, cluster_mode=cluster_mode)
+    master = df_factory.create(
+        disable_serialization_max_chunk_size=0,
+        version=1.19,
+        path=released_dfly_path,
+        cluster_mode=cluster_mode,
+    )
     replica = df_factory.create(
-        cluster_mode=cluster_mode, cluster_announce_ip=announce_ip, announce_port=announce_port
+        disable_serialization_max_chunk_size=0,
```

Comment:
dfly_version = "v1.19.2" does not support the flag, so launching the instance will fail

Reply:
in which version did you introduced this flag? maybe we can update this test to run this dfly version

Reply:
> in which version did you introduced this flag?  

I will need to check.

> in which version did you introduced this flag? maybe we can update this test to run this dfly version

There is no point because a) the bug was on that version b) we won't run the reg tests with this flag on in the future anyway

Reply:
I dont think should use this flag in regression tests so that small percentage of entries in replication/snapshot test will hit this flow.
The default for this flag will be much higher though

Reply:
> I dont think should use this flag in regression tests so that small percentage of entries in replication/snapshot test will hit this flow.

Only for extreme testing. I can disable this no issue on that